drafts/mocks/build_module_loader.go

```diff
@@ -0,0 +1,127 @@
+// Code generated by mockery v2.30.1. DO NOT EDIT.
```

Comment:
This created the new mock, but I don't believe it deleted the old mock file. I think we should delete the old mock so there's no confusion.